go/deploy/builderd/Makefile

```
@@ -93,29 +80,17 @@
 	@echo 'Targets:'
 	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "  %-20s %s\n", $$1, $$2}' $(MAKEFILE_LIST)
 
-install: build create-user ## Install the service (requires sudo)
+install: build ## Install the service (requires sudo)
```

checkmake (0.2.2)

[warning] Target body for "install" exceeds allowed length of 5 (9).

(maxbodylength)